test/Kentico.Search.Tests/Fakes/FakeSearchService.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+
+using CMS.Base;
+using CMS.DocumentEngine;
+using CMS.Search;
+
+namespace Kentico.Search.Tests
+{
+    public class FakeSearchService : SearchService
+    {
+        private string _dataSetFilePath = null;
```

Comment:
The other classes use `m` prefix for private variables. We should be consistent.